apps/src/code-studio/components/ExportDialog/Dialog.jsx

```diff
@@ -337,6 +347,16 @@ class ExportDialog extends React.Component {
     }
   }
 
+  visitExpoSite() {
+    const {expoSnackId} = this.state;
+    if (!expoSnackId) {
+      return;
+    }
+    // TODO: use new URL format once snack-web has been updated for this flow
+    // TODO: pass iconUri and splashImageUri to expo.io
```

Comment:
Are there work items for these?

Reply:
Also, for the link (below) should it redirect to the signup page? Right now, it sends me to a page with a bunch of code on it and I'm not sure what I'm supposed to do next.

Reply:
Yes, the `snack-web` changes are not yet available (that is what the TODO notes are indicating).
Once those changes are available on `snack-web`, this code will be updated to navigate to something like `/publishipa/@:username/:projectName`, which will redirect to signup/login and then explain the IPA export process.